### DISCUSSION TITLE
7.18 update except for some generting bugs

### DISCUSSION THREAD

**Discussion by hlhlhl231** [open-webui/open-webui#3967] [General]
# Bug Report
v0.3.10 - building the best open-source AI user interface.
ollama is newest.
win 11.  pinokio v1.2  

After  7.18 update except for some generting bugs.
First , i use workspace make a Personalized models. base model is qwen2:7b(GPU:4080laptop).it always pop up <Oops! There was an error in the previous response. Please try again or contact admin.>.i also go to pinokio ask ,Pinokio ruled out.Meanwhile , it also shows this.
![image](https://github.com/user-attachments/assets/cc6fedb1-4750-4598-ae03-c3c4865f32f2)

**Comment by hlhlhl231** [comment]
![image](https://github.com/user-attachments/assets/0230ef67-2cc0-4258-9701-dcdc0bf52d4c)
these are another 2 questions.

**Reply by hlhlhl231** [reply]
Finally, I can't rule out which updates have issues and which are model issues

**Reply by hlhlhl231** [reply]
Adding again, New Chat & Workspace can't be clicked.Also need refresh the browser.

**Comment by djfaizp** [comment]
same issue
